v2/terraform_flannel.py

```diff
@@ -184,10 +184,14 @@ def _deploy_ansible(self):
         self.logging.info("Succesfully deployed ansible-playbook.")
 
 
-    def _waitForConnection(self, machine, windows):
-        self.logging.info("Waiting for connection to machine %s." % machine)
+    def _waitForConnection(self, machines, windows):
+        self.logging.info("Waiting for connection to %s." % machines)
         cmd = ["ansible"]
-        cmd.append(machine)
+
+        vms = " "
+        vms = "'" + vms.join(machines) + "'"
```

Comment:
This here, very ugly. Use cmd.append("'%s'" % vms.join(machines)) like we do in L195 or L221